libjlm/include/jlm/ir/operators/store.hpp

```diff
@@ -88,12 +88,12 @@ class StoreOperation final : public jive::simple_op {
   ~StoreOperation() noexcept override;
 
   StoreOperation(
-    const PointerType & pointerType,
+    const jive::valuetype & valueType,
```

Comment:
For the LoadOperation it was called loadedType.
Why is the more generic valueType used here?

Reply:
Renamed it to storedType.